tests/temp_tests/test_colocatedJobGroup.py

```diff
@@ -35,9 +35,9 @@
 pytestmark = pytest.mark.group_a
 
 # TODO replace with LaunchSettings
```

Comment:
Looks like you can drop this `TODO` while you are here. We seemed to have moved away from having settings directly on the `Application`
```suggestion
```

Reply:
removed